app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,7 @@
+require 'csv'
+require 'pry-byebug'
+class ShowsController < ApplicationController
+    def index 
```

Comment:
Indentation is screwed up here. Make sure you install and run Rubocop (and rucocop-rspec) over your entire project

Reply:
Done